### ISSUE TITLE
Change the "updated on server" confirmation symbol

### ISSUE THREAD

**Issue by suchow** [jupyter/nbgrader#123]
I don't really recognize it. I think it's a rightward-facing arrow with a check mark? It would look cleaner if it was `glyphicon-ok` or `glyphicon-ok-sign` or `glyphicon-ok-circle`.

Actually, it looks like there's a symbol for exactly this purpose: `glyphicon-saved`.

**Comment by jhamrick** [comment]
Ok, @smeylan also said that he was confused by the symbol, so I will change it. Personally, I feel like if the current one (which is `glyphicon-floppy-saved`) is hard to parse, then some of those others will too, so maybe just `glyphicon-ok` for simplicity?

**Comment by jhamrick** [comment]
Note: the available glyphs can be seen here: http://getbootstrap.com/components/
